modules/swagger-codegen/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -22,15 +22,16 @@ using RestSharp.Portable.HttpClient;
 using RestSharp;
 {{/netStandard}}
 
-namespace {{packageName}}.Client
-{
+namespace {{packageName}}.Client {
     /// <summary>
     /// API client is mainly responsible for making the HTTP call to the API backend.
     /// </summary>
-    {{>visibility}} partial class ApiClient
-    {
-        private JsonSerializerSettings serializerSettings = new JsonSerializerSettings
-        {
+    {{>visibility}} partial class ApiClient {
+
+        private static Logger log = new Logger(nameof(ApiClient));
+        private const string SERVER_DEFAULT_ADDRESS = "{{basePath}}";
```

Comment:
Could you explain why? I thought that is completely normal to use only one place to store strings such that. In case my URL is changed, I should go step by step and replace it manually.